internal/local/app.go

```diff
@@ -35,6 +35,7 @@ func errFailedToFindApp(path string) error {
 type App struct {
 	RootDir string
 	Config  File
+	AppMeta AppMeta
```

Comment:
The issue with embedding `AppMeta` is that both `AppMeta` and `AppData` have `ConfigVersion` fields which conflict, so I'll rename this to `Meta` rather than rename the meta config version field.